daprdocs/content/en/operations/configuration/increase-read-buffer-size.md

```diff
@@ -0,0 +1,60 @@
+---
+type: docs
+title: "How-To: Handle large http read buffer size"
+linkTitle: "Http read buffer size"
+weight: 6000
+description: "Configure http read buffer size that are bigger than 4 KB"
```

Comment:
Can you please add a few sentences on why someone would want to increase the read buffer size?

Also can you please change the description to "Configure a larger http read buffer size".

Reply:
Thank you for your reply. Our company manages the current cluster application deployment by abstracting the content of the k8s yaml file into a node structure tree. The transmission data generated when a large number of applications are deployed in the upper layer is very large. Because of historical problems, temporarily The solution is to solve by setting the http read buffer size.I will modify this description to "Configure a larger http read buffer size".